cosmos-todo-list-mi/cosmos-app.yaml

```
@@ -33,18 +33,18 @@ spec:
           - name: CosmosDB__Account
             valueFrom:
               secretKeyRef:
-                name: cosmos-connection-settings  # This is the secret created by ASO associated with the CosmosDB Account. See the DatabaseAccount resource for more details.
+                name: $APP_NAME-sqldb-connection-settings  # This is the secret created by ASO associated with the CosmosDB Account. See the DatabaseAccount resource for more details.
```

Suggest including a direct link to any mentioned resources.

```suggestion
                # This is the secret created by ASO associated with the CosmosDB Account.
                name: $APP_NAME-sqldb-connection-settings  
                # See https://azure.github.io/azure-service-operator/reference/documentdb/v1api20210515/#documentdb.azure.com/v1api20210515.DatabaseAccount for more details
```